awscli/examples/medical-imaging/update-image-set-metadata.rst

```diff
@@ -0,0 +1,36 @@
+**To update image set metadata**
+
+The following ``update-image-set-metadata`` code example updates image set metadata. ::
+
+
+
+    aws medical-imaging update-image-set-metadata \
+        --datastore-id 12345678901234567890123456789012 \
+        --image-set-id ea92b0d8838c72a3f25d00d13616f87e \
+        --latest-version-id 4 \
+        --update-image-set-metadata-updates '{
```

Comment:
Needs line continuation mark for lines 11-14.

Reply:
resolved with json file.